keyboards/bestway/config.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2022 LXF-YZP(yuezp)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define RGBLIGHT_LAYERS
+#define RGBLIGHT_LAYERS_RETAIN_VAL
+#define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF
```

Comment:
These should be moved to info.json (`qmk info -kb (name) -f json` can help with this)

Reply:
> These should be moved to info.json (`qmk info -kb (name) -f json` can help with this)

Thank you for your help